plugin-server/src/worker/vm/promise-manager.ts

```diff
@@ -23,12 +26,20 @@ export class PromiseManager {
         promise.finally(() => {
             this.pendingPromises.delete(promise)
         })
+
+        this.statsd?.increment('promise_manager_promises_tracked', {
```

Comment:
Do we have a gauge metrics of the number of pending promises? Would that be useful?

Reply:
yeah that would be useful actually